forge/forge/op/loss.py

```diff
@@ -2,41 +2,41 @@
 
 # SPDX-License-Identifier: Apache-2.0
 from ..module import ForgeModule
-from .reduce import ReduceSum, ReduceAvg
-from .eltwise_unary import Exp, Log, Abs
-from .eltwise_binary import Subtract, Multiply
 from .constant import Constant
+from .eltwise_unary import Log, Abs
+from .eltwise_binary import Subtract, Multiply
+from .nn import Softmax
+from .reduce import ReduceSum, ReduceAvg
 
 
 class CrossEntropyLoss(ForgeModule):
     """
-    Cross-Entropy Loss
+    Cross-Entropy Loss (with mean reduction)
```

Comment:
I'll file an issue.

For adding the `reduction` attribute, I think it will only create unnecessary confusion... The loss module doesn't take any arguments, so the user knows that there is no option of configuration reduction (or disabling it). If I add an argument with default value "mean", then the user will think that it can change it, but he will be greeted with an assert instead. So, I won't add this now; it should be added once proper reduction configuration is added.

Reply:
Created issue #792